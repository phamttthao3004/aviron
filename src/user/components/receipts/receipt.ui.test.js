import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ReceiptUI from './receipt.ui';

describe('Receipt UI', () => {
    it('Render Receipt UI component', () => {
        const props = { data: [], onChange: undefined, onSearch: undefined, setOpenView: undefined };
        const com = shallow(<ReceiptUI {...props} />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
