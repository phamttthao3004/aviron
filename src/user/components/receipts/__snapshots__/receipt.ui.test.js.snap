// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Receipt UI Render Receipt UI component 1`] = `
<div
  id="receipt-page"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <CmsTable
        className=""
        data={Array []}
        mode="edit"
        onChange={[Function]}
        onSearch={[Function]}
        total={0}
      >
        <div
          className="cms-table-content"
        >
          <table
            className="table aviron-table action-table call-data-table"
          >
            <thead>
              <tr>
                <th>
                  #
                </th>
                <th>
                  Order date
                </th>
                <th>
                  Order number
                </th>
                <th>
                  Price
                </th>
                <th>
                  Status
                </th>
              </tr>
            </thead>
            <tbody />
          </table>
        </div>
      </CmsTable>
    </div>
  </div>
</div>
`;
